website/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "oxc-website",
+  "private": true,
+  "version": "0.0.1",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview",
+    "check": "svelte-check --tsconfig ./tsconfig.json"
+  },
+  "devDependencies": {
+    "@sveltejs/vite-plugin-svelte": "^2.0.3",
+    "@tsconfig/svelte": "^3.0.0",
+    "svelte": "^3.55.1",
+    "svelte-check": "^2.10.3",
+    "tslib": "^2.5.0",
+    "typescript": "^4.9.3",
+    "vite": "^4.2.0",
+    "vite-plugin-windicss": "^1.8.10",
+    "windicss": "^3.5.6"
```

Comment:
Considering that `windicss` is sunsetting -> (https://windicss.org/posts/sunsetting.html) I would suggest to use an alternative style solution

Reply:
Thank you for the heads up, I'll definitely look into alternatives when I get the time.